### ISSUE TITLE
Got into state where new tabs and windows are blank

### ISSUE THREAD

**Issue by ndevenish** [wez/wezterm#3992]
### What Operating System(s) are you seeing this problem on?

macOS

### WezTerm version

20230408-112425-69ae8472

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

No, and I'll explain why below

### Describe the bug

Existing tabs are fine, creating new tabs ends up with a blank screen. (below). Switching to existing tabs works fine; if I use the keyboard to close the new, blank tab, or switch to the older tabs, then the existing tabs work fine, and the tab bar along the top reappears.

The only error on the debug overlay is:
```
11:49:32.741 ERROR wezterm_gui::termwindow::render::paint > paint_pass failed: No such file or directory (os error 2)
```
![image](https://github.com/wez/wezterm/assets/529963/88f1a28f-0636-4146-8b03-3e6d51847381)

### To Reproduce

Unfortunately, closing and reopening made this go away. 

### Configuration

To avoid posting my whole (verbose) wezterm.lua, I have Removed keys, theming parts. These are the only ones that it sounds plausible that might be involved (I think this default_cwd is broken, but has been there a long while so doesn't seem the immediate cause)
```
config.default_prog = {"zsh", "-l"}
config.default_cwd = "~"

config.enable_scroll_bar = true
config.min_scroll_bar_height = "2cell"
config.scrollback_lines = 10000
config.font = wezterm.font("SF Mono", {weight = "Medium"})
config.font_size = 13
config.quit_when_all_windows_are_closed = false
config.hide_tab_bar_if_only_one_tab = true
```

### Expected Behavior

A new tab.

### Logs

```
11:05:15.820  INFO   wezterm_gui::update     > latest release 20230712-072601-f4abf8fd is newer than current build 20230408-112425-69ae8472
11:44:15.206  ERROR  wezterm_gui::termwindow::render::paint > paint_pass failed: No such file or directory (os error 2)
11:44:15.242  ERROR  wezterm_gui::termwindow::render::paint > paint_pass failed: No such file or directory (os error 2)
..... for 200 lines until end of log file....
```

### Anything else?

Unfortunately, closing and reopening made this go away. I appreciate that isn't very helpful, but thought this could at least be a "maybe something in this area of code" issue; consider this personally resolved (I will update if I manage to get it into this state again with the newer version).

**Comment by wez** [comment]
Please update to the latest release as I believe that this was a symptom of an issue resolved by 4ae176dd8a019eeaf955cc947e3789995edbed07

**Comment by ndevenish** [comment]
Plausible. I’ve updated but the previous (application) session was open a couple of weeks; and had no active “image” displayed on any of the still-existing terminal history.

I’d probably advise closing without evidence of repeat occurrence/other people running into it.
